setup.py

```diff
@@ -18,7 +18,7 @@
 with open("pennylane_orquestra/_version.py") as f:
     version = f.readlines()[-1].split()[-1].strip("\"'")
 
-requirements = ["pyyaml", "pennylane>=0.15"]
+requirements = ["pyyaml", "pennylane>=0.27"]
```

Comment:
Would we need `pennylane>=0.28` here too?

Reply:
No I think 0.27 is sufficient :)

Reply:
```suggestion
requirements = ["pyyaml", "pennylane>=0.28"]
```